src/components/common/entities/WebViewWithInjected/index.tsx

```
@@ -7,36 +7,85 @@ import {
+  useImperativeHandle,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { Animated, BackHandler, View } from "react-native";
 import WebView from "react-native-webview";
 import type { WebViewSource } from "react-native-webview/lib/WebViewTypes";
 
+type OnMessage = (
+  reqMessage: MessageEventRequestData,
+) => MessageEventResponseData | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)